packages/wdio-browserstack-service/src/service.ts

```diff
@@ -110,6 +111,17 @@ export default class BrowserstackService implements Services.ServiceInstance {
         // added to maintain backward compatibility with webdriverIO v5
         this._browser = browser ? browser : globalThis.browser
 
+        // Healing Support:
+        if (!isBrowserstackSession(this._browser)) {
+            try {
+                await AiHandler.selfHeal(this._options, caps, this._browser)
+            } catch (err) {
+                if (this._options.selfHeal === true) {
+                    BStackLogger.debug(`Error while setting up self-healing: ${err}. Disabling healing for this session.`)
```

Comment:
Shouldn't we log a customer visible error for this?

Reply:
We are adding logs only in case the selfHeal flag is set to true by the user. We should not display logs unless user wants to check the healing flow.